roles/openshift_etcd_client_certificates/tasks/main.yml

```diff
@@ -0,0 +1,7 @@
+---
+- include_role:
+    name: openshift_etcd_facts
```

Comment:
```bash
$ grep -rn "openshift_etcd_client_certificates"
roles/openshift_master/meta/main.yml:20:- role: openshift_etcd_client_certificates
roles/nuage_master/meta/main.yml:18:- role: openshift_etcd_client_certificates
roles/flannel/meta/main.yml:17:- role: openshift_etcd_client_certificates
playbooks/common/openshift-cluster/redeploy-certificates/etcd.yml:64:    - role: openshift_etcd_client_certificates
```

Before we push the etcd facts out of the role we should check its impacts in all three roles. I don't want to blindly duplicate the `openshift_etcd_facts` in all four places. Instead, I would postponed that and minimize deps of all wrappers later.

Reply:
Just call openshift_etcd_client_certificates (or whatever role + tasks after refactoring) in the etcd plays.  Most likely those are all pointing at the same hosts:  nodes and/or masters.  There's no need to run this role multiple times.